waffle_configs/bedrock-prod.env

```diff
@@ -25,13 +25,12 @@ SWITCH_EXPERIMENT_FIREFOX_NEW_TRAILHEAD=off
 SWITCH_EXPERIMENT_FIREFOX_WHATSNEW_70=off
 SWITCH_FIREFOX_ELECTION_BUILD=on
 SWITCH_FIREFOX_DEV_FALL=on
-SWITCH_FIREFOX_CONCERT_SERIES=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION=on
 SWITCH_FIREFOX_UPDATE_NOTIFICATION_MODAL=on
 SWITCH_FIREFOX_YANDEX=on
 SWITCH_FIREFOX_PRE_DOWNLOAD_NEWSLETTER=off
-SWITCH_HOME_FUNDRAISING_SEPT2019=off
```

Comment:
Deleted in bedrock with this PR: https://github.com/mozilla/bedrock/commit/712a49ede3b98abc7e2cf4b2bd73387a0e4009fa#diff-ad17ec34b227e71c15b1b11e9523b69aL45